types/htmlhint/htmlhint-tests.ts

```diff
@@ -0,0 +1,33 @@
+import { HTMLHint, Hint, Rule, Ruleset } from 'htmlhint';
```

Comment:
those are exports in original one (TS), we cannot import `R`* from module directly:
```ts
export declare const HTMLHint: HTMLHintCore;
export { HTMLRules, Reporter, HTMLParser };
```
to do this, ne shuld use direct imports from '/core/types' (?)

Reply:
I am not sure. There is no exist how to use the types but it is one of the possible solutions. What do you suggest?

Reply:
those are used e.g. by someone importing types into, say, VSCode extension, MS work arounded this following way:
https://github.com/microsoft/vscode-htmlhint/blob/master/htmlhint-server/src/typings-custom/htmlhint.d.ts
https://github.com/microsoft/vscode-htmlhint/blob/master/htmlhint-server/src/server.ts#L7
(but those are for older version)

Reply:
'htmlhint' goes via configuration, like for some postcss stuff, no direct api usage.
But that would be OK to have rules definition, so JSON schema can be quickly generated

Reply:
https://github.com/microsoft/vscode-htmlhint/issues/2